weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/utilities.ts

```diff
@@ -48,8 +49,13 @@ export const refUriToOpVersionKey = (refUri: RefUri): OpVersionKey => {
 };
 
 export const opVersionKeyToRefUri = (key: OpVersionKey): RefUri => {
-  return `${WEAVE_REF_PREFIX}${key.entity}/${key.project}/op/${key.opId}:${key.versionHash}`;
-  // return `${WANDB_ARTIFACT_REF_PREFIX}${key.entity}/${key.project}/${key.opId}:${key.versionHash}/obj`;
+  return makeRefObject(
```

Comment:
We have too much ref parsing logic floating around, centralizing to one location.